src/crimson/osd/pg.h

```diff
@@ -787,6 +787,11 @@ struct PG::do_osd_ops_params_t {
     return false;
   }
 
+  // Only used by ExecutableMessagePimpl
+  entity_name_t get_source() const {
+    return entity_name_t();
+  }
+
```

Comment:
should be orig_source_inst.name? the following member variable orig_source_inst should have the entity_name_t, right?

Reply:
Right, thanks!